### DISCUSSION TITLE
(Feature Request) PrimeNG Table - Column Resize + Reorder Unexpected Behavior

### DISCUSSION THREAD

**Discussion by RejectKidsPadawan** [primefaces/community#671] [PrimeNG] [answered]
Hello PrimeNG community,
 
I'm working on a project that uses PrimeNG tables and the desired functionality is for users to be able to both resize and reorder the columns. The current behavior shown in the attached video shows me resizing the first column (sector) and then trying to reorder it to be in the 2nd index. It reorders fine, but the resized column width seems to "stick" to the 1st index (company ends up being the resized column after the reorder).

Is there a way we could get the column resize to persist with the specific columns that were resized please (i.e "Sector" was resized to be larger and after the reorder I would like for "sector" to remain the larger / resized column)?

Tagging @RejectKid

https://github.com/primefaces/community/assets/153555953/f7410556-0cbb-4b87-804a-78d209d5d4ab

General information:
PrimeNG: 16.3.1
Node: 16.20.2
Angular CLI: 16.2.10
OS: Unbuntu 20.04

Some additional table information:
[resizableColumns]=true
[reorderableColumns]=true
[scrollable]=true
columnResizeMode=expand

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Because it resizes the first column. It doesn't matter if you move the columns, the resize action independent from column names etc due to performance reasons. It's not a bug actually, it's made by design. We're generating new styles for the table and insert them into the HTML for the performance, it's not inline style. We've added it in our internal backlog.

**Reply by RejectKidsPadawan** [reply]
Okay, thank you for the response!
